src/defaultV2-theme/blocks/mixins.scss

```diff
@@ -1,10 +1,11 @@
 @mixin page_title {
+  --default-font-size: var(--sjs-font-pagetitle-size, calc(1.5 * (var(--sjs-font-size, 16px))));
   font-family: $font-pagetitle-family;
   font-weight: $font-pagetitle-weight;
-  font-size: $font-pagetitle-size;
+  font-size: var(--default-font-size);
```

Comment:
Probably better to use different variable names for different objects. It's hard to find what variable to set or rename if they have the same names.